### ISSUE TITLE
many messages from groups that I'm not a member

### ISSUE THREAD

**Issue by gesu92** [ars3niy/tdlib-purple#150]
Hello, I have problem last time when use telegram with this plugin in pidgin, sometimes **I received many messages (and many file transference of that messages, that window get focus) of groups that I'm not in**, I check in the telegram app, and those messages of those groups are not in the app, but in pidgin get that messages. That hang up pidgin, and the group chat first appear with the name when appear that messages, but when finally stop the transfers change the name with, example: "chat-12343534543", last days that happend more frequently.
I don't known why, but seems to be groups of people of the groups that I really in, and those people share a message of those groups to my groups, I think that, but really I haven't investigated it because sometimes hang up entirely pidgin.

![Captura de pantalla_2022-02-15_18-11-44](https://user-images.githubusercontent.com/87961082/154152477-22c49619-da17-4ab3-9a94-667b6e1779e2.png)

PD= sorry my bad english.

**Comment by BenWiederhake** [comment]
That weird name usually occurs when tdlib-purple cannot fetch the name of a chat for some reason, e.g it has been converted to a supergroup, you left, or something else changed. Usually you can just close the chats and they'll never* pop up again.

EDIT: "Never" apparently isn't exactly never. It seems that getting forwarded a channel post triggers a weird bug where you receive all messages. Huh.

**Comment by gesu92** [comment]
all right, I will try just close it, I'll wait for it when pidgin hang up, thanks

**Comment by zer0def** [comment]
I've been experiencing the same thing for a number of months now - to the best of my ~~understanding~~ guesswork, these channels are "quietly subscribed to" based on post shares in conversations you're already a part of and they appear whenever there's an aggressive resync from Telegram's servers, as they're relying on client-side behavior to prune channels you're "not actually" subscribed to.

**Comment by zer0def** [comment]
Ever since bumping to a 1.8.0-supporting build from #154, this behavior seemed to have gone away.

**Comment by Thaodan** [comment]
I have the same issues since a few days with bitlbee too.

**Comment by bmomjian** [comment]
Yes, this is a serious problem if there are malicious people trying to attack your account. I installed @BenWiederhake's fork mentioned in https://github.com/ars3niy/tdlib-purple/pull/163 and am hopeful that will fix it. It would be nice to get a new packaged release so manual builds would not be necessary.
